hunting-buddy.lic

```diff
@@ -259,6 +264,17 @@ class HuntingBuddy
         echo('***STATUS*** stopping due to time')
         break
       end
+      if Flags['familiar_drag'] && @bail_on_fam
+        echo('***STATUS*** stopping due to familiar dragging from stun')
+        Flags.reset('familiar_drag')
+        break
+      end
+      if Flags['familiar_drag'] && !@bail_on_fam
+          echo('***STATUS*** heading back to room - familiar drug while stunned')
+          DRC.wait_for_script_to_complete('go2', [hunting_room])
+          #DRC.walk_to(hunting_room)
```

Comment:
❌ I think `walk_to` is a better choice here, it calls go2 internally but also handles some other cases.